OWNERS_ALIASES

```diff
@@ -10,7 +10,6 @@ aliases:
     - andreyvelich
     - gaocegege
     - johnugeorge
-    - tenzen-y
```

Comment:
You might need to remove yourself from [the `wgs.yaml`,](https://github.com/kubeflow/community/blob/master/wgs.yaml#L110) and run `make generate`